final-project/scripts/blog.js

```diff
@@ -0,0 +1,82 @@
+const $ = document.querySelector.bind(document);
+const $$ = document.querySelectorAll.bind(document);
+
+const indicatorsElement = $('.blog__indicators');
```

Comment:
Just as a recommendation, when selecting nodes with js is better to do it with selectors that are not related with styles and use a prefix, ie: .js-my-class-selector